webapp/channels/src/components/threading/global_threads/thread_list/__snapshots__/thread_list.test.tsx.snap

```diff
@@ -46,9 +46,10 @@ exports[`components/threading/global_threads/thread_list should match snapshot 1
       <div
         className="right-anchor"
       >
-        <SimpleTooltip
-          content="Mark all as read"
```

Comment:
You need to update the failing test because its trying to find content tag

Reply:
Alright, the update has been made